spec/controllers/v1/people_controller_spec.rb

```diff
@@ -59,8 +59,7 @@
         context "existing user with address info" do
 
           before do
-            FactoryGirl.create(:person, email: 'user@example.com',
-              district: FactoryGirl.create(:district))
+            FactoryGirl.create(:person, :with_district, email: 'user@example.com')
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.